Replacing "defaultVolumesToRestic" with "defaultVolumesToFsBackup" (#66083)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc

    includedNamespaces:
    - <namespace> <2>
    storageLocation: <velero-sample-1> <3>
    defaultVolumesToFsBackup: true <4>
    ttl: 720h0m0s
EOF
----
<1> `cron` expression to schedule the backup, for example, `0 7 * * *` to perform a backup every day at 7:00.
<2> Array of namespaces to back up.
<3> Name of the `backupStorageLocations` CR.

> <4> Optional:  OADP 1.2 and later: Add the `defaultVolumesToFsBackup: true` key-value pair if you are backing up volumes with Restic. OADP 1.1: Add the `defaultVolumesToRestic: true` key-value pair if you are backing up volumes with Restic.

Reviewer: It is probably the best way to handle it. It is not ideal, but it is what it is, so it is

> <4> Optional:  OADP 1.2 and later: Add the `defaultVolumesToFsBackup: true` key-value pair if you are backing up volumes with Restic. OADP 1.1: Add the `defaultVolumesToRestic: true` key-value pair if you are backing up volumes with Restic.

Reviewer 2: ```suggestion
<4> Optional: In OADP version 1.2 and later, add the `defaultVolumesToFsBackup: true` key-value pair to your configuration when performing backups of volumes with Restic. In OADP 1.1, add the `defaultVolumesToRestic: true` key-value pair when you back up volumes with Restic.
```
